scripts/docker/web/web/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM elektra/web-base:latest
+
+WORKDIR /home/elektra
+USER elektra
+
+# create start script
+RUN printf "#!/bin/bash\nkdb run-elektrad &\nkdb run-web" > start
```

Comment:
I thought you will split this up into two containers?

Please give a description about the individual dockerfiles. (as comments and in the README.md)

Reply:
@markus2330 I explained this in the PR description, `elektra/web` contains both for testing purposes (quickstart guide in README)

Reply:
I added the description to a README file

Reply:
Thank you! Please always keep all information within the repo (and not in the PR description).